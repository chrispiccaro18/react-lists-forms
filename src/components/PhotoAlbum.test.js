import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('PhotoAlbum component', () => {
  it('renders a PhotoAlbum', () => {
    const wrapper = shallow(<PhotoAlbum title="Lauren" photos={['img-string0', 'img-string1', 'img-string2']} />);
    expect(wrapper).toMatchSnapshot();
  });
});
