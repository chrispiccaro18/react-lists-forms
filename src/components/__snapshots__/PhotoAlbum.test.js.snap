// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoAlbum component renders a PhotoAlbum 1`] = `
<section>
  <h2>
    Lauren
  </h2>
  <Photos
    photos={
      Array [
        "img-string0",
        "img-string1",
        "img-string2",
      ]
    }
  />
</section>
`;
